apache/Dockerfile

```diff
@@ -102,17 +102,14 @@ RUN set -ex; \
 	tar -xf roundcubemail.tar.gz -C /usr/src/roundcubemail --strip-components=1 --no-same-owner; \
 	rm -r "$GNUPGHOME" roundcubemail.tar.gz.asc roundcubemail.tar.gz; \
 	rm -rf /usr/src/roundcubemail/installer; \
-	chown -R www-data:www-data /usr/src/roundcubemail/logs
-
-# include the wait-for-it.sh script
-RUN curl -fL https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh > /wait-for-it.sh && chmod +x /wait-for-it.sh
+	chown -R www-data:www-data /usr/src/roundcubemail/logs; \
+# Create the config dir
+	mkdir -p /var/roundcube/config
 
 # use custom PHP settings
 COPY php.ini /usr/local/etc/php/conf.d/roundcube-defaults.ini
 
 COPY --chmod=0755 docker-entrypoint.sh /
```

Comment:
Maybe move these two COPY commands up before `ENV ROUNDCUBEMAIL_VERSION=` because they are less likely to change than the roundcube sources.